packages/components/src/item-group/types.ts

```diff
@@ -0,0 +1,68 @@
+type ItemSize = 'small' | 'medium' | 'large';
+
+export interface ItemGroupProps {
+	/**
+	 * Renders borders around each items.
+	 *
+	 * @default false
+	 */
+	isBordered?: boolean;
+	/**
+	 * Renders with rounded corners.
+	 *
+	 * @default true
+	 */
+	isRounded?: boolean;
+	/**
+	 * Renders items individually. Even if `isBordered` is `false`, a border
+	 * in between each item will be still be displayed.
```

Comment:
If the above comment is true then this can be shorted to just `Renders a separator between each item`.

Reply:
I like the new proposed description! Although we probably need to first agree on the behaviour of `isBordered` first